### PR TITLE
No issue - Update Mike's email address and affiliation

### PR DESCRIPTION
**Pull request by miketaylr**
Just updating my email (6 months late...)

***
<a href="https://whatpr.org/compat/142.html" title="Last updated on Mar 22, 2021, 9:56 PM UTC (3e5bfe4)">Preview</a> | <a href="https://whatpr.org/compat/142/dde80c6...3e5bfe4.html" title="Last updated on Mar 22, 2021, 9:56 PM UTC (3e5bfe4)">Diff</a>

### CODE REVIEW

**Comment by miketaylr** [comment]
> Pending — @miketaylr has not signed up to participate

I guess I should sort that out.

**Comment by miketaylr** [comment]
> FATAL ERROR: The propdef for 'touch-action' is missing a 'Animation type' line.

Unrelated, but I can fix in this PR as well.

**Comment by miketaylr** [comment]
PTAL @foolip (thanks!)

File: compatibility.bs [hunk 1]
```diff
@@ -821,5 +822,5 @@ Special thanks to Apple and the WebKit.org blog authors for providing initial de
 
 This standard is written by
 <a href="https://miketaylr.com/">Mike Taylor</a>
-(<a href="https://www.mozilla.org/">Mozilla</a>,
-<a href=mailto:miket@mozilla.com>miket@mozilla.com</a>).
+(Google, LLC,
```
Comment (foolip) [inline compatibility.bs:825]:
```suggestion
(<a href=https://google.com/>Google</a>,
```

Following the style of https://streams.spec.whatwg.org/#acks

**Review by foolip — APPROVED** [review]

**Comment by miketaylr** [comment]
Thanks!

**Pull request merged**